### PR TITLE
When using advance search query will search using parent index from Item

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
When searching using advanced search we will now search in parents index where tree structure is stored. 

https://oat-sa.atlassian.net/browse/CON-23

### CODE REVIEW

File: src/searchModal.js [hunk 1]
```diff
@@ -176,11 +176,13 @@ export default function searchModalFactory(config) {
                 }
                 const classUri = _.map(selectedValue, 'classUri')[0];
                 const label = _.map(selectedValue, 'label')[0];
+                const uri = _.map(selectedValue, 'uri')[0]
```
Comment (bziondik) [inline src/searchModal.js:179]:
```suggestion
                const uri = _.map(selectedValue, 'uri')[0];
```

**Review by bziondik — APPROVED** [review]
Please, bump version in `package.json` and then update `package-lock.json ` by `npm i`
- [ ]  New code is covered by tests (if applicable)
- [x]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: src/searchModal.js [hunk 2]
```diff
@@ -176,11 +176,13 @@ export default function searchModalFactory(config) {
                 }
                 const classUri = _.map(selectedValue, 'classUri')[0];
                 const label = _.map(selectedValue, 'label')[0];
+                const uri = _.map(selectedValue, 'uri')[0];
                 const route = urlUtil.route('get', 'ClassMetadata', 'tao', {
                     classUri,
                     maxListSize: instance.config.maxListSize
                 });
                 $classFilterInput.val(label);
+                $classFilterInput.data('uri',uri);
```
Comment (habemuscode) [inline src/searchModal.js:185]: Space after comma for code style guidelines

**Review by habemuscode — APPROVED** [review]
Small comment from my side apart from @bziondik comments. The rest looks good

- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Pull request merged**